code/modules/mob/living/simple_animal/abnormality/zayin/wellcheers.dm

```diff
@@ -28,6 +28,17 @@
 	harvest_phrase = "<span class='notice'>The machine dispenses some clear-ish soda into %VESSEL.</span>"
 	harvest_phrase_third = "%PERSON holds up %VESSEL and lets %ABNO dispense some clear-ish soda into it."
 
+/mob/living/simple_animal/hostile/abnormality/wellcheers/PostSpawn()
+	..()
+	for(var/d in list(EAST, WEST))
+		var/turf/TF = get_step(src, d)
+		if(!istype(TF))
+			continue
+		if(!TF.is_blocked_turf(TRUE))
+			if(locate(/obj/structure/wellcheers_side_shrimp) in TF)
+				return
```

Comment:
Shouldn't this continue instead of returning? What if only one shrimp was obliterated or something

Reply:
btw I mean continue as in `continue`

Reply:
Definitely, oversight on my part, apologies